posts/2020-06-10-confronting-racism.md

```diff
@@ -0,0 +1,148 @@
+everyone. It prohibits harassment based on (among other things) race
+or ethnicity. If you are a member of an underrepresented minority and
+feel that anyone in the community is making you feel unwelcome, report
+it.
+
+Familiarize yourself with the codes of conduct in whatever
+organizations, meetings, or events you participate in. Taking time to
+digest these in advance will help guide your behavior but also to
+recognize when others may be out of line, and help you understand what
+steps you can take. If your professional or avocational circles don't
+have a code of conduct, you can begin discussions to find one that
+will suit the community.
+
+#### Intervention
+However, it must not fall to people of color to recognize and report
+harassment or unwelcome behavior. As members of a positive and
+welcoming community, we must not be passive bystanders. Learning to
+intervene constructively may take some practice.
+
+Robin DiAngelo has a list of [Silence Breakers for Whites in
+Cross-racial
+Discussions](https://robindiangelo.com/resources/attachment/silence-breakers-for-whites/)
+containing 18 different phrases you can use to intervene directly in
+an ongoing situation. These phrases can help diffuse a difficult
+situation calmly and allow people to save face and, ideally, replace a
+harmful discussion with a constructive one.
+
+The Southern Poverty Law Center has also produced some useful
+[suggestions](https://www.splcenter.org/20171005/splc-campus-guide-bystander-intervention)
+for bystander intervention. Although their document was intended for
+use on college campuses, the suggestions may be more broadly
+applicable. For instance, the SPLC points out that even if you do not
+intervene in the moment, you can document harassing behavior and
+provide it to moderators or authorities. In addition, they suggest
+that you can provide support to victims even after harassment has
+taken place.
+
+When you think about intervening, consider the feelings of the people
+you are hoping to protect. You may wish to discretely ask if they
+would like help in a situation, if there is time or a private channel
+available, especially if you intend to notify a moderator or
+supervisor.
+
+#### Education
+Those of us who are in the (white) majority owe it to ourselves to
+learn about race, racism, and the dark history behind them. We should
+also take time to reflect on our role in perpetuating racist power
+structures, and how we benefit from the repression of others. Today
+there is no shortage of books and resources to help us. Here are a few
+commonly-cited resources:
+
+* Stamped from the Beginning by [Ibrahim
+  Kendi](https://www.ibramxkendi.com/)
+* How to Be an Anti-Racist by Ibrahim Kendi
+* So You Want to Talk About Race by [Ijeoma
+  Oluo](http://www.ijeomaoluo.com/)
+* White Fragility by [Robin DiAngelo](https://robindiangelo.com/)
+* [Seeing White](https://www.sceneonradio.org/seeing-white/) (podcast)
```

Comment:
I think it's important to link to the authors, but would also consider a link to the materials. This is not a showstopper.

Reply:
I can do that. I didn't want to link directly to a business site, but if the authors have pages describing their own works, it does make sense to link to them here.